build.sh

```diff
@@ -22,6 +23,24 @@ output_path=$(realpath -m "${1}")
 shim_path=$(realpath -m "${2}")
 rootfs_dir=$(realpath -m "${3}")
 
+if [ "${args['luks']}" = 'true' ]; then
+  printf "Enter the LUKS2 password for the image: "
+  read PASSWD
+  luks_enabled=true
+  if [ "${args['arch']}" = 'arm64' ]; then
+    CRYPTSETUP=cryptsetup_arm64
```

Comment:
Variable names shouldn't be in all caps, that's inconsistent with the rest of the project.